keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -163,69 +163,49 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  No. It's expected to accelerate scheduling a pod upon preemption, and hence it won't impact
```

Comment:
1) The rollout can always fail (e.g. if there is a bug and scheduler will start crashlooping on certain conditions).
That's probably the only thing that requires mentioning explicitly.

2) For the second question just say "it's a scheduler features, so it doesn't affect already running workloads".

Reply:
Done